### PR TITLE
2.6.x

### PR DESCRIPTION
**Pull request by jzy-chitong56**
fix chinese Encoding
Add 24 player self-adaption(change all 12/13/14/15 player to self-adaption)
Recheck player colors(and now can self-adaption 24 colors)
Add Strategy and player dialog Page turning
Add translate to all language（tribute report and error report）
Add set ZOOM（-zoom1000 ,for 1.24~1.32.10 player , min 0 ,max 3000）
FIX dialog cannot change language
ai_only_mode will show MapInit report 
Fix DialogSetMessage only show once bug
if GetFirstHuman is Observer , then the player will choose language
add POWER_FOUNTAIN(only TFT), W3C have some map have POWER_FOUNTAIN 
add new Profiles(only TFT)
synchronization some file to 3.1.1
strategy change

### CODE REVIEW

**Comment by SMUnlimited** [comment]
Thanks commander improvements are most welcome. I'll review and merge soon. 

Will you be creating a similar PR for the additions you have added for 3.1 main master branch too?

**Comment by jzy-chitong56** [comment]
yes，but i change TR, i will Later  Provide (Four race  Can TR)，so now my code Not yet provide

**Comment by jzy-chitong56** [comment]
yes，but no now，i Teaching 4 race study TR,so code A little messy，Not easy to separate
I think disposable Submit

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,1129 +1,1202 @@
...
+- Computer skill level is now shown by default on AMAI players. (from [3.0.0]) 
+- Tweaked english gramatical errors for various strategy messages. (from [3.0.0]) 
+- Tweaks to Brewmaster and Alchemist skill picks. (Pixyy) (from [3.1.0]) 
+- Balance pass to all racial hero selection, skill selection and strategy selection. Mostly minor but most noticeable changes include (consult Pixyy)  (from [3.1.0]) 
+   - Night Elf increased use of warden and keeper.
+   - Reduced use of mirror image skill.
+- Strategy tweaks and optimizations  (consult Pixyy)  (from [3.1.0])
+   - Removed non-useful units from harass attacks and fixed undead harassment group numbering.
+   - Improvements to human, orc, undead and night elf dynamic strategy adjustment.
+- Strategy additional Improvements
```
Comment (SMUnlimited) [inline CHANGELOG.md:29]: Note these strategy changes are missing at the moment. I'll merge what you got anyway as I can tidy up changelog to match later.

**Pull request merged**